src/js/tabs/ports.js

```diff
@@ -162,60 +162,60 @@ TABS.ports.initialize = function (callback, scrollPosition) {
            31: 'SOFTSERIAL2'
         };
 
-        var gps_baudrate_e = $('select.gps_baudrate');
-        for (var i = 0; i < gpsBaudRates.length; i++) {
+        let gps_baudrate_e = $('select.gps_baudrate');
+        for (let i = 0; i < gpsBaudRates.length; i++) {
             gps_baudrate_e.append('<option value="' + gpsBaudRates[i] + '">' + gpsBaudRates[i] + '</option>');
         }
 
-        var msp_baudrate_e = $('select.msp_baudrate');
-        for (var i = 0; i < mspBaudRates.length; i++) {
+        let msp_baudrate_e = $('select.msp_baudrate');
+        for (let i = 0; i < mspBaudRates.length; i++) {
             msp_baudrate_e.append('<option value="' + mspBaudRates[i] + '">' + mspBaudRates[i] + '</option>');
         }
 
-        var telemetry_baudrate_e = $('select.telemetry_baudrate');
-        for (var i = 0; i < telemetryBaudRates.length; i++) {
+        let telemetry_baudrate_e = $('select.telemetry_baudrate');
+        for (let i = 0; i < telemetryBaudRates.length; i++) {
             telemetry_baudrate_e.append('<option value="' + telemetryBaudRates[i] + '">' + telemetryBaudRates[i] + '</option>');
         }
 
-        var blackbox_baudrate_e = $('select.blackbox_baudrate');
-        for (var i = 0; i < blackboxBaudRates.length; i++) {
+        let blackbox_baudrate_e = $('select.blackbox_baudrate');
+        for (let i = 0; i < blackboxBaudRates.length; i++) {
             blackbox_baudrate_e.append('<option value="' + blackboxBaudRates[i] + '">' + blackboxBaudRates[i] + '</option>');
         }
 
         let lastVtxControlSelected;
-        var ports_e = $('.tab-ports .ports');
+        const portsE = $('.tab-ports .ports');
```

Comment:
This should probably stay `_e` to be consistent with existing naming for element variables.

Reply:
We have changed this in several tabs and new code without the underscore. But I usually use the complete word `portsElement`.

Reply:
@McGiverGim You mean `portPickerElement`.

@mikeller For now I change it back to be consistent with the rest - we can always rebase again on naming conventions.